tests/resources/test.yaml

```diff
@@ -0,0 +1,21 @@
+seed: 42
+
+architecture:
+  name: experimental.soap_bpnn
+  training:
+    batch_size: 2
+    num_epochs: 1
+
+training_set:
+  systems:
+    read_from: ethanol_reduced_100.xyz
+    length_unit: angstrom
+  targets:
+    forces:
```

Comment:
just as a side comment, but if we allow arbitrary key name I think we should warn if someone tries to learn forces directly like this instead of doing it through the gradient of the energy.

Reply:
For sure, I will add the warning